elytron/src/main/java/org/wildfly/extension/elytron/ElytronSubsystemTransformers.java

```diff
@@ -149,6 +152,12 @@ private static void from16(ChainedTransformationDescriptionBuilder chainedBuilde
                 .setDiscard(DiscardAttributeChecker.UNDEFINED, KEY_STORE_ALIAS)
                 .addRejectCheck(RejectAttributeChecker.DEFINED, KEY_STORE)
                 .addRejectCheck(RejectAttributeChecker.DEFINED, KEY_STORE_ALIAS);
+        builder.addChildResource(PathElement.pathElement(DISTRIBUTED_REALM))
+                .getAttributeBuilder()
+                .setDiscard(DiscardAttributeChecker.UNDEFINED, IGNORE_UNAVAILABLE_REALMS)
+                .setDiscard(DiscardAttributeChecker.UNDEFINED, EMIT_EVENTS)
+                .addRejectCheck(new RejectAttributeChecker.SimpleRejectAttributeChecker(ModelNode.TRUE), IGNORE_UNAVAILABLE_REALMS)
+                .addRejectCheck(RejectAttributeChecker.DEFINED, EMIT_EVENTS);
```

Comment:
@lvydra just a minor but we should add a transformer test as well to SubsystemTransformersTestCase. An example of such test can bee seen in this commit for example https://github.com/wildfly/wildfly-core/commit/13e3181d4528a6b74c09792725edfac382d09c00